### PR TITLE
13058 setup save button for deis

### PR DESCRIPTION
**Pull request by CeciliaCY**
### Summary
Fixes [AB#13058](https://dcp-paperless.visualstudio.com/d36fd830-9029-4b77-b0c4-b0df2012eb98/_workitems/edit/13058) 

Setup save for DEIS

### Technical Explanation

Same as draft EAS

### CODE REVIEW

File: client/app/components/packages/deis/edit.hbs [hunk 1]
```diff
@@ -1,25 +1,42 @@
-<div class="grid-x grid-margin-x">
-  <div class="cell large-8">
+<SaveableForm
+  @model={{@package}}
+  @validators={{array (hash) (hash)}}
+  as |saveableForm|
+>
+
```
Comment (godfreyyeung) [inline client/app/components/packages/deis/edit.hbs:6]: tiny nit: no space here

File: client/app/components/packages/deis/edit.hbs [hunk 2]
```diff
@@ -1,25 +1,42 @@
...
+        <div class="cell large-8">
 
-    <section class="form-section">
-      <h1 class="header-large">
-        Draft Environmental Impact Statement (DEIS)
-        <small class="text-weight-normal">
-        {{if @package.dcpPackageversion (concat '(V' @package.dcpPackageversion ')')}}
-      </small>
-      </h1>
+            <section class="form-section">
```
Comment (godfreyyeung) [inline client/app/components/packages/deis/edit.hbs:10]: unindent two space

File: client/app/components/packages/deis/edit.hbs [hunk 3]
```diff
@@ -1,25 +1,42 @@
...
+                {{optionset 'bbl' 'boroughs' 'label' @package.project.dcpBorough}} |
+                {{optionset 'package' 'statuscode' 'label' @package.statuscode}}
+            </p>
+            </section>
+
+        </div>  
+        <div class="cell large-4 sticky-sidebar">
+            <saveableForm.PageNav>
+                <saveableForm.SaveButton
+                @isEnabled={{or @package.isDirty saveableForm.isSaveable}}
```
Comment (godfreyyeung) [inline client/app/components/packages/deis/edit.hbs:35]: indent

**Review by godfreyyeung — APPROVED** [review]
Awesome. I think in VS Code do you have your tab set to 4 spaces? In the bottom right you should be able to change it to 2 spaces. After that it would be nice if you replace all tabs with just two spaces before merging. Thanks!
image

**Comment by CeciliaCY** [comment]
@godfreyyeung Thanks! I update the space from 4 to 2.

**Comment by allthesignals** [comment]
@godfreyyeung does template lint pick up on this? wonder how we can enforce this through linting.

**Pull request merged**